server/src/test/java/io/spine/server/entity/storage/ColumnReaderTest.java

```diff
@@ -74,7 +79,9 @@ void fromClassWithColumns() {
             ColumnReader columnReader = forClass(EntityWithManyGetters.class);
             Collection<EntityColumn> entityColumns = columnReader.readColumns();
 
-            assertThat(entityColumns).hasSize(3);
+            assertThat(entityColumns).hasSize(5);
```

Comment:
Not sure why my comment on this disappeared...

I asked to rewrite this piece and avoid counting objects. Instead we can compare two `Set`s.